client/state/data-layer/analytics/adTracking.js

```diff
@@ -6,18 +6,24 @@ import { get } from 'lodash';
 /**
  * Internal dependencies
  */
-import { recordSelectedVertical } from 'lib/analytics/ad-tracking';
+import {
+	trackCustomAdWordsRemarketingEvent,
+	trackCustomFacebookConversionEvent,
+} from 'lib/analytics/ad-tracking';
 import { SIGNUP_STEPS_SURVEY_SET } from 'state/action-types';
 
-function handleSignupStepsSurveyStart( _, action, next ) {
+function recordSelectedVertical( _, action, next ) {
```

Comment:
can we maybe not use the underscore? I personally like it but in JS it irks me out and I don't think we typically use it in Calypso either. we can only have a single underscore because it's a legitimate variable name…